tools/aiecc/aiecc/cl_arguments.py

```diff
@@ -61,6 +61,16 @@ def parse_args():
             default=False,
             action='store_true',
             help='Compile using pathfinder router')
+    parser.add_argument('--aie-generate-xaie',
+            dest="xaie",
+            default=1,
+            action='store_const', const=1,
+            help='Generate libxaie v1 drivers (default is v1)')
+    parser.add_argument('--aie-generate-xaiev2',
+            dest="xaie",
+            default=1,
+            action='store_const', const=2,
+            help='Generate libxaie v2 drivers (default is v1)')
```

Comment:
I think we will probably have other targets, too...  can I suggest "--target=libxaiev1", "--target=libxaiev2" for these options?

Reply:
Good idea. I'll make that change on the next commit.